### ISSUE TITLE
IS-04 enhancement request: Enable mock registry w/o DNS-SD enabled

### ISSUE THREAD

**Issue by billt-hlit** [AMWA-TV/nmos-testing#223]
It would be useful in my environment if the IS-04 tests were to provide a registry even if DNS-SD were turned off. Aside from the convenience value of having the testing registry and the tests in a single program, some additional tests (like test_21, "Node correctly interprets a 200 code from a registry" and my proposed tests to ensure devices are registered before their resources) could be run without DNS-SD.

Does that sound worthwhile and feasible?

**Comment by garethsb-sony** [comment]
Presumably, this would require the node under test to have an out-of-band means to tell it the registry proto/IP/port/version?

**Comment by billt-hlit** [comment]
Yes, agreed. That's what I've been having to do anyhow, only I also have been running a separate registry by hand as well.
